partner_contact_in_several_companies/models/res_partner.py

```diff
@@ -23,10 +25,13 @@ class ResPartner(models.Model):
     other_contact_ids = fields.One2many('res.partner', 'contact_id',
                                         string='Others Positions')
 
-    @api.one
+    @api.multi
     @api.depends('contact_id')
     def _get_contact_type(self):
-        self.contact_type = self.contact_id and 'attached' or 'standalone'
+        for record in self:
+            record.contact_type = record.contact_id \
```

Comment:
While I agree that the existing line needed to be cleaned up, `\` is a dirty hack that should be avoided IMO. I believe the following would be functionally equivalent & read much nicer:
```
record.contact_type = 'attached' if record.contact_id else 'standalone'
```